### PR TITLE
Polishing the notebook

### PR DESCRIPTION
**Pull request by dberenbaum**
Made some minor changes to the notebook:
1. Added a step to `dvc pull` inside the notebook so it doesn't fail if you `git clone` and immediately run the notebook.
2. Put in a text description for each section.

### CODE REVIEW

File: example-get-started-experiments/code/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This is an auto-generated repository for use in [DVC](https://dvc.org)
 This is a Computer Vision (CV) project that solves the problem of segmenting out 
 swimming pools from satellite images. 
 
-![Example result](./results/evaluate/plots/images/REGION_1-24_0_1024_0_1024.png)
```
Comment (dberenbaum) [inline example-get-started-experiments/code/README.md:14]: This was pointing to a path that doesn't exist

Comment (daavoo) [reply example-get-started-experiments/code/README.md:14]: This is going to be an issue until we fix #188 , the paths are from a random rample

File: example-get-started-experiments/code/notebooks/TrainSegModel.ipynb
Comment (daavoo) [inline example-get-started-experiments/code/notebooks/TrainSegModel.ipynb:1]: Updated to pass `report="notebook"`

**Review by daavoo — APPROVED** [review]

**Comment by daavoo** [comment]
Deployed https://github.com/iterative/example-get-started-experiments/blob/main/notebooks/TrainSegModel.ipynb

**Pull request merged**